### PR TITLE
fix gcp external lb instructions

### PR DESCRIPTION
**Pull request by leena-jawale**
issue: https://issues.redhat.com/browse/AAP-14056

### CODE REVIEW

**Review by chrisahl — APPROVED** [review]
Figured it wouldn't be too far off due to HTTPS change.  Please add issue link in PR description

File: stories/topics/proc-gcp-external-load-balancer.adoc [hunk 1]
```diff
@@ -51,16 +51,21 @@ For the {HubName} load balancer, the correct instance group has the suffix `-aap
 .. In the resulting dialog box, select *USE EXISTING PORT NAME*.
 .. Set *Balancing mode* to *Rate*.
 .. Set *Maximum RPS* to 300.
+.. Scroll down until you see *Cloud CDN*. Unselect checkbox for *Cloud CDN*.
```
Comment (ianf77) [inline stories/topics/proc-gcp-external-load-balancer.adoc:54]: Unselect the checkbox

File: stories/topics/proc-gcp-external-load-balancer.adoc [hunk 2]
```diff
@@ -51,16 +51,21 @@ For the {HubName} load balancer, the correct instance group has the suffix `-aap
...
 .. For {Controllername}, use the following health check settings
-* In the *Health Check* dialog box, set *Protocol* to `TCP`, and *Port* to `8052`.
+* In the *Health Check* dialog box, set *Protocol* to `HTTPS`, and *Port* to `8052`.
 * Set *Request* to `/api/v2/ping/`.
 .. For {HubName}, use the following health check settings
-* In the *Health Check* dialog box, set *Protocol* to `TCP`, and *Port* to `8080`.
+* In the *Health Check* dialog box, set *Protocol* to `HTTPS`, and *Port* to `8080`.
 * Set *Request* to `/api/galaxy/pulp/api/v3/status/`.
-* Keep all other default settings. 
+.. Scroll down to *Health critearia* section.
```
Comment (ianf77) [inline stories/topics/proc-gcp-external-load-balancer.adoc:64]: criteria

File: stories/topics/proc-gcp-external-load-balancer.adoc [hunk 3]
```diff
@@ -51,16 +51,21 @@ For the {HubName} load balancer, the correct instance group has the suffix `-aap
...
+.. Set *Check interval* to 15.
```
Comment (ianf77) [inline stories/topics/proc-gcp-external-load-balancer.adoc:65]: Are these in particular fields?

**Review by ianf77 — COMMENTED** [review]
some comments

File: stories/topics/proc-gcp-external-load-balancer.adoc (hunk 3 above)
Comment (chrisahl) [reply stories/topics/proc-gcp-external-load-balancer.adoc:65]: So I guess from:
.. Scroll down to *Health critearia* section.
.. Set *Check interval* to 15.
to:
.. Scroll down to *Health criteria* section.
.. In the *Check interval* field, enter the value 15.

**Review by ianf77 — APPROVED** [review]
LGTM

**Pull request merged**